### DISCUSSION TITLE
Region matrix with TSS as reference point

### DISCUSSION THREAD

**Discussion by ashleshagogate** [stuart-lab/signac#1387] [Q&A] [answered]
Hello,
I have a dataset with WT cells and KO cells. I also have a list of differentially accessible peaks in KO v/s WT. I want to plot fragment counts for those DAPs centered at TSS. Is there a function in Signac like Deeptools compute matrix reference point? From my understanding the current RegionMatrix function is like the Deeptools compute matrix scale regions function.

**Comment by timoast** [comment, marked as answer]
Yes, you should be able to use the RegionMatrix and RegionHeatmap functions
